Fixtures/IncrementalBuildTests/SourceCodeError/Sources/main.swift

```diff
@@ -0,0 +1,2 @@
+print(firstLine)
+print(secondLine)
```

Comment:
Is this test package any different from `PackageError`? I couldn't spot the different but its kind hard to see in the PR review view...